.github/ISSUE_TEMPLATE/bug_fix.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Fix
+description: Create a ticket to resolve the bug.
+title: "[EPIC/SUB]  Model X (ASR/STT/LLM/TTS): suitable title"
+labels: "bug"
+assignees: ""
+---
+
+**Troubleshoot the bug**
+
+A clear concise description about what the bug is.
+
+- Bug 1
+- Bug 2
+
+**Solution**
+
+Describe your solution.
```

Comment:
Nit: `Concisely state the problem the solution addresses and perhaps a brief description of the solution itself.`